2200028/2.py

```diff
@@ -0,0 +1,19 @@
+import sys
+sys.stdin = open("예제입력.txt", "r")
+
+T = int(input())
+for t in range(1, T+1):
+    num = list(map(int,input().split()))
+
+    odd = num[0:17:2]
+    even = num[1:16:2]
+    even_result = sum(even)
+    odd_result = 0
+    for i in odd:
+        i = i*2
+        odd_result += i
+        result = 10-((int(even_result)+int(odd_result))%10)
```

Comment:
인덱스를 잘 나눠주셨네요! 👍 
```suggestion
    odd_result = 2 * sum(odd)
    result = 10-((even_result+odd_result)%10)
```
짝수 자리의 합을 구했던 것과 마찬가지로, 홀수 자리 수 2배의 합을 구하는 것도 `sum` 함수 활용이 가능합니다!
또한 `sum`함수의 반환 값은 이미 integer 이므로, `int` 함수를 다시 적용하지 않으셔도 괜찮습니다 😄